framework/include/vectorpostprocessors/SphericalAverage.h

```diff
@@ -64,11 +64,8 @@ class SphericalAverage : public ElementVectorPostprocessor
   /// value mid point of the bin
   VectorPostprocessorValue & _bin_center;
 
-  /// local thread copy of the sum vectors
-  std::vector<VectorPostprocessorValue> _sum_tmp;
-
   /// sample count per bin
-  VectorPostprocessorValue _count_tmp;
+  std::vector<unsigned int> _counts;
```

Comment:
Note to reviewer: I think we should avoid creating VectoPostprocessor instances in our objects. We may eventually want to make the constructor private like we have for Material and Variable containers to help prevent users from missing the reference in normal declarations. This particular vector is not a declared vector so I switched it to C++ vector.